### PR TITLE
Remove NativeLibraryLoader

### LINKED ISSUE #128: Remove NativeLibraryLoader paths on DNX
~~Blocked on aspnet/dnx#402~~
~~Blocked on aspnet/dnx#3119~~

### PR DESCRIPTION
**Pull request by natemcmaster**
No longer necessary as NuGet and DNX should handle native libraries in the 'runtimes' folder.

Verified on UWP. It will load from "runtimes/win7-(arch)".

Fixes #128 

Note it does not solve #134

### CODE REVIEW

File: src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs [hunk 1]
```diff
@@ -6,42 +6,11 @@
...
-#endif
 
 namespace Microsoft.Data.Sqlite.Interop
 {
     internal static class NativeMethods
     {
-#if NET451 || DOTNET5_4
-        static NativeMethods()
-        {
-            var loaded = NativeLibraryLoader.TryLoad("sqlite3");
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs]: Wait, we still need this on Full .NET! Only UWP and DNX will handle the runtimes stuff.

**Comment by natemcmaster** [comment]
@bricelam Is there a difference between the native library we pull from https://www.sqlite.org/2015/sqlite-uap-3090000.vsix vs https://www.sqlite.org/2015/sqlite-dll-win32-x86-3090000.zip?

File: src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs (hunk 1 above)
Comment (natemcmaster) [inline src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs]: Oh, I was under the impression nuget 3 would handle this. Does it not automatically bring in native libraries?

**Comment by bricelam** [comment]
Yes, the vsix is compiled for WinRT

**Comment by natemcmaster** [comment]
Jumped the :gun:. Will leave this to @bricelam as fixing #128 is fairly nuanced.

**Pull request closed without merging**